widgets/BigQueryTable-batchsink.json

```diff
@@ -136,7 +136,7 @@
           "label": "Location",
           "name": "location",
           "widget-attributes" : {
-            "placeholder": "Location of the big query dataset"
+            "placeholder": "US"
```

Comment:
is it significant that it's US here and us for gcs?

Reply:
This is to be consistent with the gcp console. In gcs, we use `us` and in BQ we use `US`